public_html/wp-content/mu-plugins/blocks/webpack.config.js

```diff
@@ -17,6 +17,8 @@ const webpackConfig = {
 		minimize: true,
 	},
 
+	devtool: 'production' === NODE_ENV ? 'none' : 'cheap-module-eval-source-map',
```

Comment:
It feels odd that we have lines 5, 14, and 20 all doing essentially the same thing w/r/t setting the environment var. I'd be ideal to have it set once (in 5), and then be able to use that in `14` and `20`, e.g.:

```js
mode: NODE_ENV,
devtool: 'production' === mode ? 'none' : 'cheap-module-eval-source-map',
```

Does that seem right to y'all as well?

I'm not sure that line 20 can reference `mode` though, since it won't be fully defined yet? Or maybe it needs to use `this.mode` or something?

Reply:
Opened https://github.com/WordPress/wordcamp.org/issues/37 for this